backend/Magefile.go

```diff
@@ -11,6 +11,7 @@ import (
 // See https://github.com/grafana/grafana-plugin-sdk-go/blob/002de0b2b925ac930952dc380682100dee27b439/build/common.go#L108.
 var cb build.BeforeBuildCallback = func(cfg build.Config) (build.Config, error) {
 	cfg.EnableCGo = true
+	// cfg.EnableDebug = true
```

Comment:
needs to be enabled in order to build without optimisation flags for a 'better debugging experience'

Reply:
😁